apollos-church-api/src/data/wcc-features/index.js

```diff
@@ -30,9 +31,28 @@ class WCCFeatures extends baseFeatures.dataSource {
     __typename: 'SocialIconsFeature',
   });
 
+  async mediaSeries({ seriesId } = {}) {
+    const { WCCSeries } = this.context.dataSources;
+    const item = await WCCSeries.getFromId(seriesId);
```

Comment:
@vinnyjth this line is returning an error. I can't figure out why. It looks like a scoping issue, but I can't find the why. `.paginate` calls work just fine, which seems to use the same constructs and scopes. If you have ~30min to timeblock looking at this, I would appreciate it. I'll jump back in it but I'm ultimately stuck right now.

Reply:
![image](https://user-images.githubusercontent.com/103927/81084675-d01fe080-8ebb-11ea-9e57-5b9c27a5bc0f.png)

Reply:
🤦 figured out the issue!

`WCCSeries.getFromId(522)` vs `WCCSeries.getFromId('522')`